xtask/src/bootstrap_groth16.rs

```diff
@@ -26,8 +30,15 @@ use risc0_zkvm::{
 };
 use risc0_zkvm_methods::{multi_test::MultiTestSpec, MULTI_TEST_ELF, MULTI_TEST_ID};
 
-#[derive(Parser)]
-pub struct BootstrapGroth16;
+#[derive(Debug, Parser)]
+pub struct BootstrapGroth16 {
+    /// ris0-ethereum repository path
+    #[arg(long, env, default_value = "../risc0-ethereum")]
```

Comment:
I feel like this should have a default. This util doesn't need to be super simple to use, and it seems better to me to be explicit where it is being invoked. That and avoiding assumption about the existence of paths outside the repo whenever possible.
```suggestion
    #[arg(long, env)]
```